packages/tezos-requester/tezos-requester.8.1/opam

```diff
@@ -20,6 +20,7 @@ build: [
   ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 synopsis: "Tezos: generic resource fetching service"
+available: false
```

Comment:
This field already exists above in this file

Reply:
I removed the duplicates